### ISSUE TITLE
SSH Connection fails with "key too small" error

### ISSUE THREAD

**Issue by lostminds** [Sequel-Ace/Sequel-Ace#476]
- Sequel Ace Version: 2.2.2
- macOS Version: 10.15.7
- MySQL Version: 5.5.37

**Description**
I'm trying to transition from Sequel Pro to Sequel Ace and I'm copying over my favorites from Sequel Pro (it would have been great if there was a quick import for this). They are all SSH connections to various servers (with SSH host, user and password), for various databases. In one case however, I cannot connect to the server when entering the same details in Sequel Ace as I did in Sequel Pro. Instead I get the following error:

```
Unable to connect to host _*hostname here*_, or the request timed out.

Be sure that the address is correct and that you have the necessary privileges, or try increasing the connection timeout (currently 10 seconds).

MySQL said: SSL connection error: error:141A318A:SSL routines:tls_process_ske_dhe:dh key too small
```

**Steps To Reproduce**
1. Enter database and SSH connection details in the favorites connection interface
2. Click Try connection or Connect

**Expected Behaviour**
Connection to the server, interface change to the database access.

**Is Issue Present in [Latest Beta](https://github.com/Sequel-Ace/Sequel-Ace/releases)?**
Same problem in latest beta.

**Additional Context**

**Comment by Kaspik** [comment]
Re: Export / Import - https://github.com/Sequel-Ace/Sequel-Ace/issues/155

**Comment by lostminds** [comment]
@Kaspik Ah, nice with the export. A bit too late for me though as I've now manually copied them over field by field already. If I had known about it and found how to do it I would probably have tried it though. Maybe show a hint about this possibility when you start up and don't have any Favorites yet?

**Comment by Kaspik** [comment]
There is a Get started doc (https://github.com/Sequel-Ace/Sequel-Ace/blob/main/docs/get-started/index.md) as well as website (https://sequel-ace.com/get-started/) - We can come up with something else, but it would be pretty down the priorities right now... :)

**Comment by lostminds** [comment]
Sure, I understand. The issue I'm reporting here for example is now preventing me from starting to use Sequel Ace as I can't connect at all to a third of the databases I need to work with.

**Comment by Kaspik** [comment]
Sure, I believe someone else more "specialized" on this will reply, but searching for your issue finds for example this: https://stackoverflow.com/a/36417794/4927614

TLDR: Issue might be your server using weak DH Key as we have newer OpenSSL library than 3 years outdated Sequel Pro does, thus OpenSSL itself rejects the connection.

**Comment by Jason-Morcos** [comment]
We don't support MySQL versions older than 5.6. Unfortunately we had to drop support for these older versions to support MySQL 8. Please see compatibility notes on the repo main page. https://github.com/Sequel-Ace/Sequel-Ace

Sequel Ace 2.0.2 was the last version to support MySQL 5.5. You can install that, but we're not offering much support for it anymore. https://github.com/Sequel-Ace/Sequel-Ace/releases/tag/2.0.2
